cmd/tuf/main.go

```diff
@@ -124,11 +125,16 @@ func parseExpires(arg string) (time.Time, error) {
 	return time.Now().AddDate(0, 0, days).UTC(), nil
 }
 
-func getPassphrase(role string, confirm bool) ([]byte, error) {
-	if pass := os.Getenv(fmt.Sprintf("TUF_%s_PASSPHRASE", strings.ToUpper(role))); pass != "" {
+func getPassphrase(role string, confirm bool, change bool) ([]byte, error) {
```

Comment:
Two boolean flags feels a little opaque, but I can't easily think of anything better without a big refactor. Maybe the options pattern to get named args? But that seems like overkill here.

Reply:
Indeed, I wanted to do a less invasive way 👍